src/pages/docker/deploy/developer-mode.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Developer mode
+description: Start the Docker environment in developer mode.
+---
+
+# Developer mode
+
+Developer mode supports an active development environment with full, writable file system permissions. This option builds the Docker environment in developer mode and verifies configured service versions.
+
+On macOS and Windows systems, performance is slower in developer mode because of additional file synchronization operations. However, you can improve performance by using either the `manual-native` or the `mutagen` file synchronization option when you generate the `docker-compose.yml` file. See [Synchronizing data in Docker](../setup/synchronize-data.md).
+
+<InlineAlert variant="info" slots="text"/>
+
+The `ece-tools` package supports developer mode starting with version 2002.0.18.
+
+Large files (>1 GB) can cause a period of inactivity. DB dumps and archive files—ZIP, SQL, GZ, and BZ2—are not necessary to sync. You can find exclusions to these file types in the `mutagen.sh` file.
+
+**Prerequisites:**
+
+-  Complete the [installation steps](../setup/initialize-docker.md).
+-  [Install file synchronization tools](../setup/synchronize-data.md) if needed.
+
+**To launch the Docker environment in developer mode**:
+
+1. In your local project root, generate the Docker Compose configuration file. You can use the service configuration options, such as `--php`, to [specify a version](../containers/index.md#service-configuration-options).
+
+   ```bash
+   ./vendor/bin/ece-docker build:compose --mode="developer"
+   ```
+
+   <InlineAlert variant="info" slots="text"/>
```

Comment:
The `InlineAlert` block doesn't currently work in lists ([DEVSITE-359](https://jira.corp.adobe.com/browse/DEVSITE-359)). 

Commenting them out will remove the blank note box but serve as a placeholder that you can search for and uncomment when the devsite team fixes the issue.